src/compiler/utilitiesPublic.ts

```diff
@@ -76,10 +76,11 @@ import {
     getElementOrPropertyAccessArgumentExpressionOrName,
     getEmitScriptTarget,
     getJSDocCommentsAndTags,
+    getJSDocRoot,
     getJSDocTypeParameterDeclarations,
     hasAccessorModifier,
-    hasDecorators,
     HasDecorators,
+    hasDecorators,

```

Comment:
What re-sorted this? Hoping to pin down these so we don't keep flipping back and forth.

Reply:
I think `lint --fix` does that

Reply:
Confusing, given it doesn't complain when it's sorted a specific way...

Reply:
I think to reproduce this issue need to do the following steps
1. Add some items to the file (`length`, `getJSDocRoot` etc.)
2. Run **QF** to add non-imported items
3. Run `lint --fix`